packages/api-client/src/components/SectionFilterButton.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+const { selected, controls } = defineProps<{
+  selected?: boolean
+  controls?: string | undefined
+}>()
+</script>
+<template>
+  <button
+    :aria-controls="controls ?? ''"
+    :aria-selected="!!selected"
```

Comment:
```suggestion
    :aria-selected="Boolean(selected)"
```
nitpick doesn't really matter